src/main/resources/banner.txt

```diff
@@ -0,0 +1,31 @@
+tasks.withType(Checkstyle) {
```

Comment:
I don't this this file should have this content. banner.txt is used for displaying service name in console while running

Reply:
fixed